public_html/app/Helpers/SessionHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+session_start();
+
+function flash($name = '', $message = '', $class = 'alert alert-success'){
+    if(!empty($name)){
```

Comment:
wow nice nesting, you should refactor this function

Reply:
"wow" :DD